packages/polytechcolloscopeclient/lib/polytechcolloscopeclient.dart

```diff
@@ -0,0 +1,3 @@
+export 'src/colloscope_datastructs.dart';
```

Comment:
They are all named model in the repo, maybe keeping a more uniform vocabulary is better.

Reply:
Sure